WeatherMax.py

```diff
@@ -0,0 +1,148 @@
+def tempLists(filName):
+    with open(filName, 'r') as f:
+        minT = 0
+        dateList = []
+        maxList2 = []
+        bList = []
+        minList3 = []
+        meanHumList = []
+        mContents = []
+        mContents = f.readline()
+        i = 1
+        while i <= 31:
```

Comment:
Can you explain what we are doing with this loop of 31 iterations?

Reply:
reading the whole file

Reply:
I was looking for the end of file function but could not find it. That's why I gave a hard coded value.

Reply:
Readlines is going to return an interator. You can iterate over it in order to read lines. Secondly we are not sure we are going to get 31 or less or more lines so we can't make it work that way.

Reply:
I have edit the variable names and function names

Reply:
And did you pushed those changes? @saad107